build.gradle

```diff
@@ -18,16 +18,16 @@ configurations {
     provided
 }
 
-version = "0.3.12"
+version = "0.3.12.2"
 
 sourceCompatibility = 1.7
 targetCompatibility = 1.7
 
 dependencies {
-    compile  "org.embulk:embulk-core:0.8.15"
-    provided "org.embulk:embulk-core:0.8.15"
-    compile  "org.embulk.base.restclient:embulk-base-restclient:0.5.2"
-    compile  "org.embulk.base.restclient:embulk-util-retryhelper-jetty92:0.5.2"
+    compile  "org.embulk:embulk-core:0.8.24"
+    provided "org.embulk:embulk-core:0.8.24"

```

Comment:
I'm not sure if using Embulk v0.8.24 is safer nor not.
It was released yesterday and embulk-base-restclient is using v0.8.18